src/mrack/utils.py

```diff
@@ -198,13 +263,15 @@ def get_shortname(hostname):
 
 def get_username(host, meta_host, config):
     """Find username from sources db/metadata/config."""
-    username = host.username or meta_host.get("username")
-    default_user = get_config_value(config["users"], meta_host["os"])
-
+    provider = host.provider.name
+    dict_key = meta_host["os"]
+    default_user = None
     if is_windows_host(meta_host):
-        default_user = default_user or "Administrator"
+        default_user = "Administrator"
```

Comment:
#NITPICK  I would add newline here so optically we can see if branch has only default user set but that change should be considered optional.
```suggestion
        default_user = "Administrator"
    
```